packages/sdk/base/package.json

```diff
@@ -7,7 +7,7 @@
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
   "sideEffects": false,
-  "homepage": "https://celo-sdk-docs.readthedocs.io/en/latest/base",
+  "homepage": "https://docs.celo.org/developer/tools#celo-sdk-reference-docs",
```

Comment:
(for future reference)

We'll need to make sure the links at this URL point to the correct directory in `celo-org/developer-tooling`

```md
You can find the reference documentation for all of the [Celo SDK packages](https://github.com/celo-org/celo-monorepo/tree/master/packages/sdk) found in the celo monorepo here. 
```

Source: [tools.md](https://github.com/celo-org/docs/blob/3a782f7328d2fbb6d49805423a5fd58371ba6564/docs/developer/tools.md?plain=1#L56)

```md
<PageRef url="https://celo-sdk-docs.readthedocs.io/" pageName="Celo SDK Reference Docs" />
```

Source: [tools.md](https://github.com/celo-org/docs/blob/3a782f7328d2fbb6d49805423a5fd58371ba6564/docs/developer/tools.md?plain=1#L78)